control/iosys.py

```diff
@@ -450,6 +450,10 @@ def issiso(self):
         """Check to see if a system is single input, single output"""
         return self.ninputs == 1 and self.noutputs == 1
 
+    def isstatic(self):
```

Comment:
statespace and xferfcn have methods `is_static_gain()` I added. should we choose a standard conventiuon? I would lean toward isstatic because it is shorter and matches `issiso()`,

Reply:
I agree we should be uniform.  I'm not sure that `is_static_gain`() would make sense for a nonlinear system, so that is another reason to opt for `isstatic()`.  

Also: these are mainly internally used functions, so I think it is OK to stick with `isstatic`/`issiso` rather than `is_static`/`is_siso`.

Reply:
Converted to `_isstatic` in latest push since this really is an internal function, not intended to be seen by the user.

@sawyerbfuller: when we update `is_static_gain`, we can either change to `_isstatic` if all usage is internal or change to `isstatic` (and update `descfcn.py` for consistency).

Reply:
renamed in #524